core/Plugin/ControllerAdmin.php

```diff
@@ -271,15 +278,23 @@ private static function notifyWhenPhpVersionIsEOL()
             return;
         }
 
-        $notifyPhpIsEOL = Piwik::hasUserSuperUserAccess() && ! self::isPhpVersionAtLeast71();
+        $notifyPhpIsEOL = Piwik::hasUserSuperUserAccess() && self::isPhpVersionEOL();
         if (!$notifyPhpIsEOL) {
             return;
         }
 
         $deprecatedMajorPhpVersion = PHP_MAJOR_VERSION . '.' . PHP_MINOR_VERSION;
-        $message = Piwik::translate('General_WarningPiwikWillStopSupportingPHPVersion', array($deprecatedMajorPhpVersion, self::getNextRequiredMinimumPHP()))
-            . "<br/> "
-            . Piwik::translate('General_WarningPhpVersionXIsTooOld', $deprecatedMajorPhpVersion);
+
+        $message = '';
+
+        if (version_compare(PHP_VERSION, self::getNextRequiredMinimumPHP(), '<')) {
+            $message = Piwik::translate(
+                'General_WarningPiwikWillStopSupportingPHPVersion',
```

Comment:
That part should currently not be shown at all. Guess it had been there in case the next release will require a higher version than the current minimal version.